R/irods-demo.R

```diff
@@ -207,11 +206,12 @@ dry_run_irods <- function(user, pass, recreate) {
 
 # look up table for irods_demo images
 irods_images_ref <- c(
-  "irods_demo_irods-catalog",
-  "irods_demo_irods-catalog-provider",
-  "irods_demo_irods-client-icommands",
-  "irods_demo_irods-client-rest-cpp",
-  "irods_demo_nginx-reverse-proxy"
+  "irods_demo-irods-catalog",
+  "irods_demo-irods-catalog-provider",
+  "irods_demo-irods-client-icommands",
+  "irods_demo-irods-client-rest-cpp",
+  "irods_demo-nginx-reverse-proxy",
+  "irods/irods_http_api"
```

Comment:
this doesn't seem correct...

Reply:
oh... because irods_demo is not building its own image (but rather using the one already published), this is the container name for the running container?

i had expected to see `irods_demo-irods-client-http-api`